include/exadg/time_integration/time_integration_constants_base.h

```diff
@@ -70,6 +71,19 @@ class TimeIntegratorConstantsBase
   print(dealii::ConditionalOStream & pcout) const = 0;
 
 protected:
+  /**
+   * Can be used to fill the unused components of the vector with zeros.
+   * This is needed since current_order might be smaller than order, e.g.,
+   * when using start_with_low_order = true
+   */
+  void
+  zero_out_unused_constants(unsigned int const first_idx, std::vector<double> & constants)
```

Comment:
I do not like the name `first_idx`. The previous name `current_order` was more intuitive in my opinion, also regarding comment https://github.com/exadg/exadg/pull/469/files#r1217691459.

The naming `..._unused_constants()` is somewhat misleading. If I remember correctly, we zero out exactly for the reason that the constants are indeed "used", but that the respective entries of the vector should not have an effect in order to get correct results (which we realize by multiplication of a vector by 0). Hence, a naming like `switch_off_high_order_constants()` or `disable_high_order_constants()` would be much more intuitive in my opinion.

Reply:
[a63b519](https://github.com/exadg/exadg/pull/469/commits/a63b51958000e5041f91ccbb314b126ec95e8b48)